addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -65,7 +65,7 @@ Text.prototype.oDeleteBackward = function (offset, alreadyMoved = false) {
     setSelection(parentNode, firstSplitOffset);
 };
 
-HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false) {
+HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false, maxSizeNodeToMove = null) {
```

Comment:
That's a confusing new argument. Could you document it? Is that simply to avoid deleting blocks? If so, why?